section2/reflection.md

```diff
@@ -1,15 +1,32 @@
+- A conditional statement is a code block that instructs the computer system to determine whether to do one thing or another within a set of criteria using "if statements". These use operators that will tell the computer which one thing or the other to do when determining the outcome. There are also data types that choose whether the set is true or false based on the subsequent variables and values given to the code block for print.
+- One daily life conditional could be a door lock. If a door is unlocked, it will open with or without a key. If the door is unlocked and you do not have a key, it will not open. If the door is unlocked but you have a key, it will open.
+- A conditional that is probably used in the Spotify app: If the logged in user has a premium subscription, there will be no advertisements played. If the logged in user has a premium subscription, downloaded music will be available offline.
 
-5. How do you add multiple conditions to an `if` statement? In your own words, explain how to program reads them and determines what to do.
+### 5. How do you add multiple conditions to an `if` statement? In your own words, explain how to program reads them and determines what to do.
+- You add multiple conditions to an `if` statement by also utilizing `else if` and `else` functions. The program reads `else if` as in an alternative input if the first condition does not apply and will output that "second option" so-to-say. Additionally, it will read `else` in the event that none of the other `else if` functions apply to the variables and values indicated within the code. Anything else that is inputted will output what is indicated by the `else` function. Automatically the first indication will always be what is indicated by the `if` statement prior to moving onto the `else if` then subsequently the `else` and whatever corresponds to each by the log.console(output).
 
-6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code?
+### 6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code?
+- To determine that two conditions are met, I can use the logical operator &&
+- To determine that 1 of 2 conditions are met, I can use the logical operator ||
 
-7. What questions do you still have about `if` statements and/or functions?
\ No newline at end of file
+### 7. What questions do you still have about `if` statements and/or functions?
+- Is there a way to have a code that runs with multiple conditions that will exclude a specific item?
+  - For example, is it possible to create an `if` statement that can determine "if all of these statements are true BUT x, then y".
```

Comment:
Yes this would probably get you into nested conditionals OR multiple conditionals in a specific order AND OR use of logical operators.